study.md

```diff
@@ -24,7 +24,11 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A SPA is a web application that exists on one page i.e. the URL never changes.
+The interactivity comes from changing the state of the one page, not from
+switching to different pages.
+
+Used knowledge I had from before.
```

Comment:
You can also change the url based on state changes to enable the use of the back button and such